front/src/modules/SimulationReportSheet/Route.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable no-nested-ternary */
+import { Table, TR, TH, TD } from '@ag-media/react-pdf-table';
+import { View, Text } from '@react-pdf/renderer';
+import { useTranslation } from 'react-i18next';
+
+import type { OperationalPointWithTimeAndSpeed } from 'applications/operationalStudies/types';
+import type { StdcmSuccessResponse } from 'applications/stdcm/types';
+
+import styles from './styles/SimulationReportStyleSheet';
+import formatRouteTable from './utils/formatRouteTable';
+
+type RouteProps =
+  | { mode: 'stdcm'; stdcmData: StdcmSuccessResponse }
+  | { mode: 'operationalStudies'; operationalPointsList: OperationalPointWithTimeAndSpeed[] };
+
+const RouteTable = (props: RouteProps) => {
+  const { t } = useTranslation('stdcm');
+  const { mode } = props;
+  const isStdcm = mode === 'stdcm';
+
+  const rows = formatRouteTable({ ...props });
+
+  return (
+    <View style={styles.consistAndRoute.stopTableContainer}>
+      <Table style={styles.consistAndRoute.stopTable}>
+        <TH style={styles.consistAndRoute.stopTableTH}>
+          <View style={styles.consistAndRoute.stopTableIndexWidth}>
+            <TD aria-label="line-count" />
+          </View>
+          <View style={styles.consistAndRoute.stopTableOpWidth}>
+            <TD>{t('reportSheet.operationalPoint')}</TD>
+          </View>
+          <View style={styles.consistAndRoute.stopTableChWidth}>
+            <TD>{t('reportSheet.code')}</TD>
+          </View>
+          <View style={styles.consistAndRoute.stopTableEndWidth}>
+            <TD>{t('reportSheet.endStop')}</TD>
+          </View>
+          {isStdcm && (
+            <View style={styles.consistAndRoute.stopForWidth}>
+              <TD>{t('reportSheet.stopTime')}</TD>
+            </View>
+          )}
+          <View style={styles.consistAndRoute.stopTableStartWidth}>
+            <TD>{t('reportSheet.startStop')}</TD>
+          </View>
+          <View style={styles.consistAndRoute.stopTableStopTypeWidth}>
+            {isStdcm && <TD>{t('reportSheet.stopType')}</TD>}
+          </View>
```

Comment:
You should include the view in the if too
```suggestion
          {isStdcm && (
            <View style={styles.consistAndRoute.stopTableStopTypeWidth}>
              <TD>{t('reportSheet.stopType')}</TD>
            </View>
          )}
```

Reply:
If I do this, the columns on the simulation sheet for the operational studies are all out of line. 
With the current code (which is also the original code), everything is aligned correctly, and the “stopType” column only appears on STDCM, as it should be

Reply:
Ok thanks for the precision !